CMakeLists.txt

```diff
@@ -211,6 +211,19 @@ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin)
 set_property(GLOBAL PROPERTY USE_FOLDERS ON)
 set_property(GLOBAL PROPERTY PREDEFINED_TARGETS_FOLDER "CMake")
 
+# option to enable precompiled headers
+sfml_set_option(SFML_ENABLE_PCH FALSE BOOL "TRUE to enable precompiled headers for SFML builds -- only supported on Windows/Linux")
+
+if(SFML_ENABLE_PCH AND SFML_OS_MACOS)
+    message(FATAL_ERROR "Precompiled headers are currently not supported in macOS builds")
+endif()
```

Comment:
Can we leave the option (and error) at the original position as well?

Reply:
Yes, will move it back.